### PR TITLE
Nudge Component

### PR DESCRIPTION
**Pull request by ShahRohan27**
## List of props
```
 id: string;
  destroy: () => void;
  heading: string;
  supportText: string;
  duration?: number;
  cta1?: string;
  cta2?: string;
  cta1OnClick?: any;
  cta2OnClick?: any;
  leftImage: React.ReactNode;
  rightImage: React.ReactNode;
  className?: string;
  position: "top-right" | "top-bottom";
```

## Storybook screenshots
Screenshot 2023-02-01 at 3 05 50 PM
Screenshot 2023-02-01 at 3 06 06 PM
Screenshot 2023-02-01 at 3 06 18 PM

### CODE REVIEW

**Review by aadeshkulkarni-fynd — APPROVED** [review]

File: src/base/common.scss [hunk 1]
```diff
@@ -33,6 +34,7 @@ body {
   display: flex;
   justify-content: space-evenly;
   flex-wrap: wrap;
+  gap: 0.4em;
```
Comment (Zissan) [inline src/base/common.scss:37]: Why are we using **_em_**

File: src/components/Nudge/Nudge.scss [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    z-index: $ZIndex3;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &-wrapper {
+    display: flex;
+    flex-direction: column;
+    background-color: $ColorSecondaryGrey80;
+    border-radius: 2.4em;
```
Comment (Zissan) [inline src/components/Nudge/Nudge.scss:18]: em?

File: src/components/Nudge/Nudge.test.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            duration: 1000,
+          })
+        }
+      >
+        Show Nudge
+      </button>
+    );
+    const ShowNudge = screen.getByText("Show Nudge");
+    userEvent.click(ShowNudge);
+    delay(1000).then(() => {
```
Comment (Zissan) [inline src/components/Nudge/Nudge.test.tsx:75]: async await can be used

File: src/components/Nudge/NudgeManager.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  cta2?: string;
+  cta1OnClick?: Function;
+  cta2OnClick?: Function;
+  leftImage: React.ReactNode;
+  rightImage: React.ReactNode;
+  className?: string;
+  position: "top-right" | "top-bottom";
+}
+
+export class NudgeManager {
```
Comment (Zissan) [inline src/components/Nudge/NudgeManager.tsx:23]: I would recommend to use React Portal and HOC's for the same

**Pull request merged**